src/scene/scenegraphnode.cpp

```diff
@@ -595,6 +652,91 @@ void SceneGraphNode::setDependencies(const std::vector<SceneGraphNode*>& depende
     }
 }
 
+void SceneGraphNode::computeScreenSpaceData(RenderData& newData) {
+    // Purposely slow the update rate of screen space position in order to reduce the
+    // effects of jittering in the position of information icon markers in web gui.
+    auto now = std::chrono::high_resolution_clock::now();
+    if ((now - _lastUpdate) < std::chrono::milliseconds(100)) {
+        return;
+    }
+    _lastUpdate = now;
+
+    // Calculate ndc
+    const Camera& cam = newData.camera;
+    const glm::dvec3& worldPos = _worldPositionCached;
+    const glm::dvec4 clipSpace = glm::dmat4(cam.projectionMatrix()) *
+                                 cam.combinedViewMatrixNoScale() * glm::vec4(worldPos, 1.0);
```

Comment:
The method `combinedViewMatrixNoScale` seems to be have introduced only for this use case. Is it really needed? At first glance, it seems that a scaling shouldn't matter here... I'm not sure and would need to think about this a little more.

Reply:
Removed combinedViewMatrixNoScale() and replaced it with combinedViewMatrix()